pkg/controller/statefulset/stateful_set.go

```diff
@@ -52,6 +52,10 @@ var controllerKind = apps.SchemeGroupVersion.WithKind("StatefulSet")
 // podKind contains the schema.GroupVersionKind for pods.
 var podKind = v1.SchemeGroupVersion.WithKind("Pod")
 
+const StatefulSetIndex = "statefulSetName"
+
+var OrphanPodIndexKey = "_ORPHAN_POD"
```

Comment:
Drop it, you've introduced this in #130859

Reply:
Sorry, missed these after rebasing. Was in a rush.
Please check this PR https://github.com/kubernetes/kubernetes/pull/130961 also when you get a chance.